At least QWebSettings and QWebElement are not exported in libQt5WebKit.dll.a
but are needed by Qt5WebKitWidgets

--- origsrc/qtwebkit-opensource-src-5.2.0/Source/api.pri	2013-12-08 11:10:13.000000000 -0600
+++ src/qtwebkit-opensource-src-5.2.0/Source/api.pri	2014-01-06 05:13:52.308646800 -0600
@@ -46,6 +46,7 @@ load(webkit_modules)
 # Resources have to be included directly in the final binary for MSVC.
 # The linker won't pick them from a static library since they aren't referenced.
 win* {
+    LIBS += -Wl,--export-all-symbols
     RESOURCES += $$PWD/WebCore/WebCore.qrc
     include_webinspector {
         # WEBCORE_GENERATED_SOURCES_DIR is defined in WebCore.pri, included by
